### PR TITLE
Update news for next release

### PR DESCRIPTION
**Pull request by papajulio**

### CODE REVIEW

**Comment by blaxter** [comment]
:+1:

File: NEWS [hunk 1]
```diff
@@ -1,3 +1,14 @@
+master
+------
+
+Enhancements
+ - Complete contact properties saving between profiles
```
Comment (sixstone-qq) [inline NEWS]: For me this line makes no sense. Where is the verb?

Comment (exekias) [inline NEWS]: I think it is "Complete", the same way you say "Fix" on the next lines

Comment (blaxter) [inline NEWS]: -------------------------------^ there 
and ----------------------------------------------------------^ there 
:trollface: 

Probably something like "More contact's fields are saved" will be better. The "saving between profiles" thing is a way (that probably only developer will understand) to refer that something is stored in sogo's database.

**Comment by papajulio** [comment]
@sixstone-qq @blaxter @exekias  what do you think now? Better?

**Comment by blaxter** [comment]
@papajulio much better

File: NEWS [hunk 2]
```diff
@@ -1,3 +1,14 @@
+master
+------
+
+Enhancements
+ - Full contact properties working and saved between Outlooks
```
Comment (sixstone-qq) [inline NEWS]: - Missing contact fields are now saved and available when sharing it
  (Missing contact fields here)

**Pull request merged**